docs/nodes/nodes-library/trigger-nodes/GitlabTrigger/README.md

```diff
@@ -0,0 +1,30 @@
+---
+permalink: /nodes/n8n-nodes-base.gitlabTrigger
+---
+
+# GitLab Trigger
+
+[GitLab](https://gitlab.com/) is a web-based DevOps lifecycle tool that provides a Git-repository manager providing wiki, issue-tracking and continuous integration/continuous deployment pipeline features, using an open-source license, developed by GitLab Inc.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Gitlab/README.md).
+:::
+
+
+## Example Usage
+
+This workflow allows you to receive updates for Gitlab events. You can also find the [workflow](https://n8n.io/workflows/528) on the website. This example usage workflow would use the following node.
+- [Gitlab Trigger]()
+
+The final workflow should look like the following image.
+
+![A workflow with the Gitlab Trigger node](./workflow.png)
+
+
+### 1. Gitlab Trigger node
+
+1. First of all, you'll have to enter credentials for the Gitlab Trigger node. You can find out how to do that [here](../../../credentials/Gitlab/README.md).
+2. Enter the repository owner in the *Repository Owner* field.
+3. Enter the repository name in the *Repository Name* field.
+4. Select the events you want to receive updates for from the *Events* dropdown list.
```

Comment:
To keep things consistent - can you please direct the user to select * here? You'll have to update the workflow, description, title of the workflow on n8n.io and so on. 

This is a good example to follow: https://docs.n8n.io/nodes/n8n-nodes-base.chargebeeTrigger/#_1-chargebee-trigger-node